web/vtadmin/src/components/routes/tablet/Advanced.tsx

```diff
@@ -0,0 +1,242 @@
+import React, { useState } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { deleteTablet, reparentTablet, startReplication, stopReplication } from '../../../api/http';
+import { vtadmin } from '../../../proto/vtadmin';
+import { isPrimary } from '../../../util/tablets';
+import { Icon, Icons } from '../../Icon';
+import { success, warn } from '../../Snackbar';
+import { TextInput } from '../../TextInput';
+
+interface AdvancedProps {
+    tablet: vtadmin.Tablet | undefined;
+}
+
+interface RouteParams {
+    alias: string;
+    clusterID: string;
+}
+
+const Advanced: React.FC<AdvancedProps> = ({ tablet }) => {
+    const { clusterID, alias } = useParams<RouteParams>();
+    const history = useHistory();
+    const primary = isPrimary(tablet);
+    // DeleteTablet
+    const [typedAlias, setTypedAlias] = useState('');
+    const [deleteLoading, setDeleteLoading] = useState(false);
+    const onDeleteTablet = async () => {
```

Comment:
I think [mutations](https://react-query.tanstack.com/guides/mutations) would be a perfect fit for all of these operations (delete, reparent, start/stop replication). 

They also let us do cool stuff like query invalidation + optimistic updates. (We could, for example, do a whole thing where we optimistically update a deleted tablet in the UI as "deleting", since you're totally right that the tablet takes foreverrr to disappear. I thought it was maybe a query caching thing, but it seems like Vitess is doing "topo stuff". Definitely out of scope for this PR, but it would be super cool!!) 

FWIW I have a couple of mutations in my hacked-up demo branch: https://github.com/vitessio/vitess/compare/main...tinyspeck:sarabee-demo-experiments Caveat that they're typed as `any` types since... well, demo branch :grimacing: BUT I do recall mutations being kind of a pain to type correctly... if it gives you trouble, feel free to throw some `any` types on there and we can always tidy it up later. :)

Reply:
Oh yeah! I'll change these all to use mutation instead.